### PR TITLE
Add pedantic, dart --fix -w, and a few manual fixes.

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

**Comment by domesticmouse** [comment]
PTAL @kevmoo

File: analysis_options.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+include: package:pedantic/analysis_options.1.7.0.yaml
```
Comment (kevmoo) [inline analysis_options.yaml]: Add a newline at the end

Comment (domesticmouse) [reply analysis_options.yaml]: done

**Comment by domesticmouse** [comment]
Now with newlines and less lint warnings.

```bash
brettmorgan-macpro1:rpc brettmorgan$ pub run test
02:41 +154: All tests passed!                                                                                                
```

**Pull request closed without merging**